From d32cdb74f9277dc7c63507fae5f8fe3b0fd713d7 Mon Sep 17 00:00:00 2001
From: Tejas Patel <tejas.patel1@einfochips.com>
Date: Fri, 21 Jan 2022 10:17:19 +0530
Subject: [PATCH 07/13] arch: arm64: boot: dts: Update CSI1 freq of IMX8MQ

This patch updates the frequency of MIPI CSI1 of IMX8MQ, to get
4 data lane working.

Change-Id: Ife474a52240142ab55bcec1437ae85cc78776a4b
Signed-off-by: Tejas Patel <tejas.patel1@einfochips.com>
Signed-off-by: Tanvi Chauhan <tanvi.chauhan@einfochips.com>
---
 arch/arm64/boot/dts/freescale/imx8mq.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/imx8mq.dtsi b/arch/arm64/boot/dts/freescale/imx8mq.dtsi
index 8cd660a22..a434ed697 100755
--- a/arch/arm64/boot/dts/freescale/imx8mq.dtsi
+++ b/arch/arm64/boot/dts/freescale/imx8mq.dtsi
@@ -1139,7 +1139,7 @@
 				assigned-clocks = <&clk IMX8MQ_CLK_CSI1_CORE>,
 						  <&clk IMX8MQ_CLK_CSI1_PHY_REF>,
 						  <&clk IMX8MQ_CLK_CSI1_ESC>;
-				assigned-clock-rates = <133000000>, <100000000>, <66000000>;
+				assigned-clock-rates = <266000000>, <333000000>, <66000000>;
 				power-domains = <&pgc_mipi_csi1>;
 				csis-phy-reset = <&src 0x4c 7>;
 				phy-gpr = <&iomuxc_gpr 0x88>;
-- 
2.17.1

